apps/src/applab/designMode.js

```diff
@@ -105,6 +105,26 @@ designMode.resetElementTray = function (allowEditing) {
 };
 
 /**
+ * If the filename is relative (contains no slashes), then prepend
+ * the path to the assets directory for this project to the filename.
+ * @param {string} filename
+ * @returns {string}
+ */
+designMode.maybeAddAssetPathPrefix = function (filename) {
+  filename = filename || '';
+  if (filename.indexOf('/') !== -1) {
+    return filename;
+  }
+
+  var channelId = dashboard && dashboard.project.current && dashboard.project.current.id;
+  if (!channelId) {
+    return filename;
```

Comment:
I'm open to alternatives. the goal is to avoid a javascript error until we fix the bug where design mode sometimes doesn't have a channel id. this code should go away once we always have a channel id.